src/components/ResultItem/ResultItem.tsx

```diff
@@ -1,17 +1,32 @@
 import React from 'react';
 
+type meta = {
+  type?: string;
+  date?: Date;
+  lastUpdated?: Date;
+};
+
 type ResultItemProps = {
   className?: string;
   href: string;
-  meta?: string;
+  meta?: meta;
   text?: string;
   title: string;
 };
 
-export const ResultItem = ({ meta, title, href, text }: ResultItemProps) => (
+export const ResultItem = ({
+  meta: { type, date, lastUpdated },
+  title,
+  href,
+  text
+}: ResultItemProps) => (
   <div className="result-item">
     <div className="result-item__meta">
-      <span className="result-item__meta-type">{meta}</span>
+      <span className="result-item__meta-type">{type}</span>
```

Comment:
className should be `result-item__meta--type` (double hyphen)

Reply:
not necessarily

if it's a modifier of result-item__meta then yes use double hyphen

if it's related to result-item__meta as it is then ...meta-type is appropriate

result-item__meta__type is just daft 🙄 (effectively it couples the markup and the style - less flexible)